api/src/main/java/org/openmrs/PatientIdentifier.java

```diff
@@ -54,30 +67,43 @@ public class PatientIdentifier extends BaseChangeableOpenmrsData implements java
 	/**
 	 * @since 1.5
 	 */
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "patient_identifier_id")
```

Comment:
```suggestion
	@Id
	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "patient_dentifier_id_seq")
	@GenericGenerator(
		name = "patient_dentifier_patient_dentifier_id_seq",
		strategy = "native",
		parameters = @Parameter(name = "sequence", value = "patient_identifier_patient_identifier_id_seq")
	)
	@Column(name = "patient_identifier_id")
```

change ID generation to `sequence` as used in xml file

Reply:
@ManojLL thank you for your good suggestion. let me fix it